src/submission/Nav.jsx

```diff
@@ -26,7 +30,7 @@ export default class EditsNav extends Component {
       upload: {
         isReachable: () => true,
         isErrored: () => this.props.code === PARSED_WITH_ERRORS,
-        isCompleted: () => this.props.code > VALIDATING,
+        isCompleted: () => this.props.code > PARSED_WITH_ERRORS,
```

Comment:
We should be able to say upload `isCompleted` at `PARSED_WITH_ERRORS`, make sense?

Reply:
I don't think that's true, cause they should be stuck on the upload page until validation is complete